addons/MountMuzzle/constants.lua

```diff
@@ -99,13 +100,47 @@ muzzles = {
         name = 'zone',
         song = music.songs.zone,
         description = 'Current Zone Music'
+    },
+    custom = {
+        name = 'custom',
+        song = music.songs.custom,
+        description = 'User-Defined Per-Mount Music'
     }
 }
 
+songs = {
```

Comment:
Is this the same list that `setbgm` uses? If so, it would be nice to combine them somewhere. Honestly, this looks like something for resources (probably added in the `fixes.xml`). But shouldn't stop this PR, just something to do in the future, so that we don't have to maintain both lists.

Reply:
@z16 
There may be an argument for them to be one addon instead of two since, they both simply change game music. :shrug:

Reply:
I haven't read the PR enough to remember if it actually does it, but there were talks of making this addon also able to select a *random* BGM to go along with the random mount. Setbgm wouldn't be able to do that without some refactoring. Besides, associating a specific different song to each mount also feels out of scope for setbgm, but ymmv.

Anyway, +1 to adding custom background music resources. I can even take care of that!

Reply:
This is indeed the same list as setbgm. I'm not at all opposed to combining them and/or moving the track list into resources, etc.